python/ray/serve/_private/logging_utils.py

```diff
@@ -279,6 +279,7 @@ def configure_component_logger(
     component_type: Optional[ServeComponentType] = None,
     max_bytes: Optional[int] = None,
     backup_count: Optional[int] = None,
+    for_default_logger: bool = False,
```

Comment:
This is an abstraction leak. `configure_component_logger` doesn't need to know about the "default logger," instead we should make this flag directly control the desired behavior:
```suggestion
    stream_handler_only: bool = False,
```